static-assets/components/cstudio-common/common-api.js

```diff
@@ -7780,14 +7803,14 @@ var nodeOpen = false,
                         "</tr>"].join("");
                 }
 
-                if(lockOwner && lockOwner != null && lockOwner.trim() != "") {                    
+                if(lockOwner && lockOwner != null && lockOwner.trim() != "") {
```

Comment:
`lockOwner && lockOwner != null` are redundant conditions unless lockOwner is a boolean
avoid potential type coercion, use `!==`.